pkg/ui/ccl/src/views/clusterviz/containers/map/modalLocalities.tsx

```diff
@@ -83,107 +70,42 @@ class NodeBox extends React.Component<NodeBoxProps, any> {
 
-class ModalTreeNodeView extends React.Component<ModalTreeNodeViewProps, ModalTreeNodeViewState> {
-  constructor(props: any) {
-    super(props);
-    this.state = {
-      inTransition: false,
-    };
-  }
-
-  onRest = () => {
-    this.setState({inTransition: false});
-  }
-
-  shouldAnimate() {
-    return this.state.inTransition || this.props.mode === "tree";
-  }
-
-  componentWillReceiveProps(newProps: ModalTreeNodeViewProps) {
-    if (this.props.mode !== newProps.mode) {
-      this.setState({inTransition: true});
-    }
-  }
-
+class ModalTreeNodeView extends React.Component<ModalTreeNodeViewProps, any> {
   renderContents = (b: BoxDimensions) => {
-    const { node, mode } = this.props;
+    const { node } = this.props;
     const box = new Box(b.x, b.y, b.w, b.h);
-    const showChildren = mode === "tree" && !this.state.inTransition;
     return node.isLocality()
-      ? <LocalityBox
-          box={box}
-          mode={mode}
-          showChildren={showChildren}
-          data={node}
-          projection={this.props.projection}
-        />
+      ? <LocalityBox box={box} data={node} />
       : <NodeBox box={box} data={node.data as SimulatedNodeStatus} />;
   }
 
   render() {
-    const { node, mode } = this.props;
-
-    // Determine the coordinates of this components bounding box based on the
-    // current display mode; either its position in the computed tree map,
-    // or its position on the world map.
-    let coords: BoxDimensions;
-    switch (mode) {
-      case "map":
-        const center = this.props.projection(node.longLat());
-        coords = {x: center[0] - 50, y: center[1] - 50, w: 100, h: 100};
-        break;
-      default:
-        coords = {x: node.x, y: node.y, w: node.dx, h: node.dy};
-    }
-
-    const targetStyle = this.shouldAnimate()
-      ? _.mapValues(coords, n => spring(n, { stiffness: 250, precision: 1 }))
-      : coords;
-    return (
-      <Motion
-        defaultStyle={coords}
-        style={targetStyle}
-        onRest={this.onRest}
-      >
-        {this.renderContents}
-      </Motion>
-    );
+    const { node } = this.props;
+
+    const center = this.props.projection(node.longLat());
+    let coords: BoxDimensions = {x: center[0] - 50, y: center[1] - 50, w: 100, h: 100};
+
+    return this.renderContents(coords);
   }
 }
+
 interface ModalLocalitiesViewProps {
   nodeHistories: SimulatedNodeStatus[];
   projection: d3.geo.Projection;
   zoom: ZoomTransformer;
 }
 
 export class ModalLocalitiesView extends React.Component<ModalLocalitiesViewProps, any> {
-  constructor(props: any) {
-    super(props);
-    this.state = {
-      mode: "map",
-    };
-  }
-
-  toggleMode = () => {
-    const mode = this.state.mode === "tree" ? "map" : "tree";
-    this.setState({ mode });
-  }
-
   render() {
     const localities = new LocalityCollection();
     this.props.nodeHistories.forEach(nh => localities.addNode(nh));
     localities.computeTreeMapLayout(this.props.zoom.viewportSize());
```

Comment:
looks like this call should be removed if we're removing the TreeMap. Also the method `computeTreeMapLayout` should be removed.

Reply:
Yep, there's also a handful of comments that I noticed I missed, too.

Reply:
Oh nvm, those comments were left over from my previous pr, not this one.